lib/pages/completion/completion_page.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/extensions/build_context_ext.dart';
+import 'package:flutter_survey/gen/assets.gen.dart';
+import 'package:flutter_survey/gen/colors.gen.dart';
+
+const _DELAY_TIME = 3;
+
+class CompletionPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    Future.delayed(Duration(seconds: _DELAY_TIME), () async {
+      Navigator.of(context).popUntil((route) => route.isFirst);
+    });
+
+    return Container(
+      color: ColorName.blackRussian,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          Expanded(child: SizedBox.shrink()),
+          Assets.images.bgCompletion.image(),
```

Comment:
FYI: this completion icon will be replaced by animation in https://github.com/luongvo/flutter-survey/issues/23